ddtrace/contrib/django/patch.py

```diff
@@ -100,6 +110,9 @@ def set_connection(wrapped, instance, args, kwargs):
     if not isinstance(django.db.connections.__setitem__, wrapt.ObjectProxy):
         django.db.connections.__setitem__ = wrapt.FunctionWrapper(django.db.connections.__setitem__, set_connection)
 
+    for conn in django.db.connections.all():
```

Comment:
🤔 is this change supposed to be here?

Reply:
old branch/bad merge from master, since we did remove this "recently" right?

Reply:
So the reason for this was mainly because of problems with testing multiple db connections. But if you try and start a setup the django app with multiple db backends, our patching does not get applied to all connections:

```
>>> import django
>>> django.setup()
>>> django.db.connections.__setitem__
<FunctionWrapper at 0x103febf40 for method at 0x103fb4580>
>>> django.db.connections["postgres"].cursor
<bound method BaseDatabaseWrapper.cursor of <django.db.backends.postgresql.base.DatabaseWrapper object at 0x103dfe4f0>>
>>> django.db.connections["default"].cursor
<FunctionWrapper at 0x104003ee0 for method at 0x103ff3ec0>
``` 

I can pull this change out into a separate PR

Reply:
Oh interesting, yeah makes sense to pull into a separate PR IMO